v2/tailnet_settings.go

```diff
@@ -0,0 +1,73 @@
+package tsclient
+
+import (
+	"context"
+	"net/http"
+)
+
+// TailnetSettingsResource provides an API to view/control settings for a tailnet.
```

Comment:
I wonder if we should start linking to api docs from some of these comments

Reply:
Seems worthwhile.